### PR TITLE
Add tenant user endpoint

### PR DESCRIPTION
**Pull request by jrpedrianes**

### CODE REVIEW

File: specs/api.kuflow.com/v2022-10-08/openapi.yaml [hunk 1]
```diff
@@ -61,6 +61,12 @@ tags:
       description: Find out more about our principal model.
       url: "https://docs.kuflow.com/fine-grained-concepts/users-groups-roles"
     x-displayName: Principal
+  - name: tenantUser
+    description: Operations about tenant users.
+    externalDocs:
+      description: Find out more about our principal model.
+      url: "https://kuflow.com"
```
Comment (7eben) [inline specs/api.kuflow.com/v2022-10-08/openapi.yaml:68]: I think is better the doc url, this is for devs

File: specs/api.kuflow.com/v2022-10-08/openapi.yaml [hunk 2]
```diff
@@ -2067,17 +2174,6 @@ components:
...
-        - objectType
-      discriminator:
-        propertyName: objectType
-        mapping:
-          "AUTHENTICATION": "#/components/schemas/Authentication"
-          "PROCESS": "#/components/schemas/Process"
-          "PROCESS_PAGE_ITEM": "#/components/schemas/ProcessPageItem"
-          "TASK": "#/components/schemas/Task"
-          "TASK_PAGE_ITEM": "#/components/schemas/TaskPageItem"
-          "WORKER": "#/components/schemas/Worker"
```
Comment (7eben) [inline specs/api.kuflow.com/v2022-10-08/openapi.yaml:2080]: Is not needed to mark as deprecated?

File: specs/api.kuflow.com/v2022-10-08/openapi.yaml (hunk 1 above)
Comment (jrpedrianes) [reply specs/api.kuflow.com/v2022-10-08/openapi.yaml]: :+1:

File: specs/api.kuflow.com/v2022-10-08/openapi.yaml (hunk 2 above)
Comment (jrpedrianes) [reply specs/api.kuflow.com/v2022-10-08/openapi.yaml:2080]: Not, in this case only we need to remove the discriminator section and mark the "objectType" property as deprecated.

**Review by 7eben — APPROVED** [review]

**Pull request merged**